src/test/java/nextstep/courses/domain/PaidEnrollmentPolicyTest.java

```diff
@@ -0,0 +1,66 @@
+package nextstep.courses.domain;
+
+
+import nextstep.payments.domain.Payment;
+import nextstep.users.domain.NsUser;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
+
+class PaidEnrollmentPolicyTest {
+
+  private PaidEnrollmentPolicy policy;
+  private Session session;
+  private NsUser user;
+  private Course course;
+
+  @BeforeEach
+  void setUp() {
+    course = new Course("백엔드 과정", 1L);
+    policy = new PaidEnrollmentPolicy(2, 10000L);
+    session = new Session(course, "test", null, null, policy);
+    user = new NsUser(1L, "user1", "pw", "홍길동", "hong@test.com");
+  }
+
+  @Test
+  void 정원이_남아있고_결제금액이_일치하면_통과() {
+    Payment payment = new Payment("id", session, user, 10000L);
+    policy.checkEnrollAvailability(session, payment);
+  }
+
+  @Test
+  void 정원이_초과되면_예외발생() {
+    session = new Session(1L, course, "test", LocalDateTime.now(), LocalDateTime.now(), SessionStatus.RECRUITING, new PaidEnrollmentPolicy(2, 10000L), new ArrayList<>(List.of(
```

Comment:
```suggestion
    Session session = new Session(1L, course, "test", LocalDateTime.now(), LocalDateTime.now(), SessionStatus.RECRUITING, new PaidEnrollmentPolicy(2, 10000L), new ArrayList<>(List.of(
```

만약 `session`이라는 정보가 테스트마다 달라야한다면 사실 클래스 전역으로 선언한 session은 의미가 없을 수 있을 것 같아요.

개인적으로 "공통"으로 모아둔 것은 모든 테스트가 공통적으로 사용하는 것에 대해서만 선언해야 사용하는데 혼동이 없다고 생각하는 편이에요 😄

Reply:
오 요건 제가 로직을 잘못 짰던 것 같아요!
수정했습니다 :)